test/utils/setupBabel.js

```diff
@@ -1,4 +1,11 @@
 require('@babel/register')({
   extensions: ['.js', '.ts', '.tsx'],
-  ignore: [/node_modules\/(?!@mui\/monorepo)/],
+  ignore: [
```

Comment:
Feel free to suggest how to cleanly replicate a solution that would work with pnpm `node_modules` structure, where path might be
```
/node_modules/.pnpm/github.com+material-ui@<hash>/node_modules/@mui/monorepo
```
instead of
```
/node_modules/@mui/monorepo
```
But I also find the function approach a bit more readable. 🤷

Reply:
migrate to `vitest` 😂

Reply:
Get rid of the monorepo dependency :)

Reply:
> What about this one?

@alexfauquette if others agree that it is readable enough, we can go with it. I don't have a strong opinion here. 🤷 

> migrate to `vitest` 😂

Baby steps, @Janpot. 👍 
Would love to... Honestly, our testing setup causes me headaches every time I write tests. 🙈